modules/snooker/src/main/ts/ephox/snooker/selection/Selections.ts

```diff
@@ -0,0 +1,21 @@
+import { TableSelection } from '@ephox/darwin';
+import { Element } from '@ephox/sugar';
+import { SelectionType, SelectionTypes } from './SelectionTypes';
+import { getSelectionStartCellOrCaption } from './TableSelection';
+import { Node } from '@ephox/dom-globals';
+
+export interface Selections {
+  get: () => SelectionType;
+}
+
+// tslint:disable-next-line:variable-name
+export const Selections = (bodyElm: Element<any>, start: () => Element<Node>, selectedSelector: string) => {
```

Comment:
The `body` type here probably needs to be `Element<HTMLElement>` instead or even `Element<DomElement>`. Would also probably make more sense to call it `root` instead of `body`.

Reply:
It also might be clearer if `start` was called `getStart` or something similar.